src/shared/components/modlog.tsx

```diff
@@ -906,16 +906,39 @@ export class Modlog extends Component<ModlogRouteProps, ModlogState> {
               <option value={"ModLockPost"}>Locking Posts</option>
               <option value={"ModFeaturePost"}>Featuring Posts</option>
               <option value={"ModRemoveComment"}>Removing Comments</option>
-              <option value={"ModRemoveCommunity"}>Removing Communities</option>
+              <option value={"ModLockComment"}>Locking Comments</option>
+              <option value={"AdminRemoveCommunity"}>
+                Removing Communities
+              </option>
               <option value={"ModBanFromCommunity"}>
                 Banning From Communities
               </option>
-              <option value={"ModAddCommunity"}>Adding Mod to Community</option>
+              <option value={"ModAddToCommunity"}>
+                Adding Mod to Community
+              </option>
               <option value={"ModTransferCommunity"}>
                 Transferring Communities
               </option>
-              <option value={"ModAdd"}>Adding Mod to Site</option>
-              <option value={"ModBan"}>Banning From Site</option>
+              <option value={"ModChangeCommunityVisibility"}>
+                Changing Community visibility
+              </option>
+              <option value={"AdminAdd"}>Adding Admin to Site</option>
+              <option value={"AdminBlockInstance"}>
+                Blocking a federated instance
+              </option>
+              <option value={"AdminAllowInstance"}>
+                Allowing a federated instance
+              </option>
```

Comment:
Capitalization is inconsistent between these. Compare "Changing Community visibility" to "Purging a Community" for example. I think capitalizing all words except for articles ("a", "the", "an", etc.) and prepositions ("on", "to", etc.) makes sense and is consistent with what we already have.

Also, not necessarily needed for this PR, but we should make translations for these in i18n.

Reply:
Def true, one of our oldest issues: #31